component/loki_workaround.libsonnet

```diff
@@ -30,6 +30,19 @@ local missing_metrics_token =
     type: 'kubernetes.io/service-account-token',
   };
 
+// Patches ClusterRole `logging-application-logs-reader` to aggregate to `admin`.
+local patch_app_logs_reader =
```

Comment:
I'd prefer to just copy the role instead of the more complicated detour with patch operator. Sometimes a little copying is better than a little dependency.

Not a deal breaker for me but i strongly believe it to be a more simple and maintainable way.